src/server/ntpcontroller.cpp

```diff
@@ -81,8 +99,3 @@ void NtpController::setConnmanProperty(QString key, QString value)
                   value.toStdString().c_str());
```

Comment:
This will probably fail to compile with debug logs enabled (key/value variables are left in setConnmanProperty)